packages/core/lib/console.js

```diff
@@ -138,7 +139,7 @@ class Console extends EventEmitter {
     });
   }
 
-  runSpawn(inputStrings, options, callback) {
+  runSpawn(inputStrings, options) {
```

Comment:
Want to mark this and the other methods here as being `async` (explicitly)?

Reply:
I don't think this method is `async` here, should it be?